### PR TITLE
Adjustment for Vetter2020daily

### PR DESCRIPTION
**Pull request by hww228**

### CODE REVIEW

File: data/vetter2020daily/vetter2020daily [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+    return dumper.represent_scalar(u'tag:yaml.org,2002:str', data, style='|')
+
+yaml.add_representer(folded_str, folded_str_representer)
+yaml.add_representer(literal_str, literal_str_representer)
+
+#read the data;
+df_1 = pd.read_csv("msphere.00827-20-st002.csv")
+
+#The patient information was obtained from the Challenger et al. (2022)
+patient_info = {
```
Comment (YWAN446) [inline data/vetter2020daily/vetter2020daily:28]: Please update the age based on Table 1 in Vetter et al. (2020), https://journals.asm.org/doi/10.1128/msphere.00827-20. In the markdown file, please explain this either in text blocks or comment in Python code block.

**Review by YWAN446 — COMMENTED** [review]
The YAML file looks great! One last comment.

**Comment by YWAN446** [comment]
Hi Youwei, you are almost done with your first project! Please see some of my edits and let me know if you have any questions. Once you review my changes, we can ask Fiona to review this PR. Then, we should be able to close it and merge it to the main branch.

**Review by tillahoffmann — COMMENTED** [review]
This looks great! A final question: It looks like the file `CombinedDataset.csv` is not used here?

Demographic information (ID, Age and Sex)  is extracted from `CombinedDataset.csv`. But regarding 'Age', I used the data from original paper, there is a discrepancy between original paper and `CombinedDataset.csv`.

**Comment by YWAN446** [comment]
I removed the CombinedDataset.csv as it was not used in this final version.

**Review by YWAN446 — APPROVED** [review]
Great work, Youwei! Your first PR will be closed.

**Pull request merged**